geolib/models/dsheetpiling/settings.py

```diff
@@ -33,12 +33,12 @@ def get_stage_type_from_method(
 class ModelType(IntEnum):
     """Represents the model types from D-Sheet Piling using the same integer values used in the application.
 
-    The Model type determines the kind of calculation that's performed on the construction.
+    The Model type determines the kind of construction: single pile or wall.
+    Note that the model type WALL includes sheet piling, wooden sheet piling and diaphragm wall.
```

Comment:
do not forget to change the comment as well when you change WALL back to SHEET_PILING